ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -4,24 +4,24 @@ presubmits:
     always_run: true
     branches:
     - master
-    context: ci/prow/e2e-aws
+    context: ci/prow/e2e-aws-ovn-kubernetes
```

Comment:
I don't think you want to overwrite the original test, right?

Reply:
@stevekuznetsov for the ovn-kubernetes repo's tests, yes we do since it's kinda pointless to run the aws-e2e test using openshift-sdn network plugin?